### PR TITLE
Alpine: php7 moved to community repo

### PR DESCRIPTION
**Pull request by bryanlatten**
- Memcached added to unsupported in alpine build

### CODE REVIEW

**Comment by mannytoledo** [comment]
Nice 👍

File: container/root/goss.alpine.yaml [hunk 1]
```diff
@@ -36,8 +36,8 @@ package:
     installed: true
   php7-msgpack:
     installed: true
-  php7-memcached:
-    installed: true
+  # php7-memcached:
```
Comment (digitalyuki) [inline container/root/goss.alpine.yaml:39]: Wouldn't you just remove these lines? Or is that commented out in the idea that soon it will come right back?

Comment (digitalyuki) [reply container/root/goss.alpine.yaml:39]: Based on @bryanlatten 's verbal comment it'll probably come right back as soon as it's stable, I'm ok with that.

**Comment by digitalyuki** [comment]
LGTM

**Pull request merged**